src/WpfMath/TexFormulaParser.cs

```diff
@@ -471,7 +471,13 @@ internal static SourceSpan ReadElement(SourceSpan value, ref int position)
                             formula.TextStyle,
                             environment.CreateChildEnvironment());
                         source = value.Segment(start, position - start);
-                        return new FractionAtom(source, numeratorFormula.RootAtom, denominatorFormula.RootAtom, true);
+                        return Tuple.Create(
```

Comment:
Be consistent: use `new Tuple<>` here, too (I'm doing it to avoid any artifical casts as much as possible).